.travis.yml

```diff
@@ -7,3 +7,15 @@ install:
     - pip install tox tox-travis
 script:
     - tox
+deploy:
+  provider: pypi
+  user: "lukas-bednar"
```

Comment:
please provide your credentials in case you want to publish release under your account.
Here is guide how to encrypt password: https://docs.travis-ci.com/user/deployment/pypi/